default.nix

```
@@ -565,7 +597,7 @@ rec {
       recurseForDerivations = true;
     };
 
-  inherit (nixpkgs) drun wabt wasmtime wasm nix-update;
+  inherit (nixpkgs) /* drun */ wabt wasmtime wasm nix-update;
```

```suggestion
  inherit (nixpkgs) wabt wasmtime wasm nix-update;
```
Yeah `drun` is already in scope